README.md

```diff
@@ -0,0 +1,65 @@
+# Letter Generator for datenanfragen .de
+
+> This repository contains the source code for Letters Generation (plain text or PDF) from templates  
```

Comment:
The letters are generated in accordance with DIN 5008. That is a core feature, so it would be nice to mention it in the description.